CMakeLists.txt

```diff
@@ -2138,7 +2138,7 @@ if("${CMAKE_SYSTEM_NAME}" STREQUAL "Windows")
 	")
 
 	include(CPack)
-elseif("${CMAKE_SYSTEM_NAME}" STREQUAL "Linux")
+else()

```

Comment:
It looks like this code is all Linux-specific files, it doesn't seem very relevant to macOS